Sesstion3/Assignment1/TestAssignment2.java

```diff
@@ -0,0 +1,18 @@
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+
+/**
+ * @author vinod patel
+ */
+public class TestAssignment2{
+
+	@Test
+	public void testgcd(){
+		int expResult		 = Assignment2.gcd(54,24);
```

Comment:
write Junit for all these three scenario as mentioned in assignments. gcd(2, 1)
gcd(12, 18)
gcd(100, 3)